transformer_engine/jax/cpp_extensions/custom_call.py

```diff
@@ -18,7 +19,15 @@
     pass
 
 for _name, _value in transformer_engine_jax.registrations().items():
-    xla_client.register_custom_call_target(_name, _value, platform="CUDA")
+    if "_ffi" in _name:
+        if jax_version_meet_requirement():
+            # TODO: add prep, init, exec
+            # traits = 1 (i.e. COMMAND_BUFFER_COMPATIBLE) enabled cudaGraph
+            xla_client.register_custom_call_target(
+                _name, _value, platform="CUDA", api_version=1, traits=1
```

Comment:
Would be great to add a IntEnum for `api_version` or at least a comment to describe it.
```
class CustomCallAPIVersion(IntEnum):
    OPAQUE = 0
    FFI = 1
```
```
xla_client.register_custom_call_target(
   _name, _value, platform="CUDA",
   api_version=CustomCallAPIVersion.FFI.value)
```